cpu/cpu.go

```diff
@@ -0,0 +1,2 @@
+// Package cpu implements the Gameboy Z80-like cpu.
+package cpu
```

Comment:
It's customary to have a `file.go` that matches the outer folder name`file`

So in this case, our outer folder (package name) is called  `cpu`, so its customary to have a default file called `cpu.go`.

In side all `.go` files in that folder, they all must have the same text at the top of the files declaring
`package <yourpackagenamehere>`

It's also customary to have this package name match the folder name.